Functions/xASL_bids_BIDS2Legacy.m

```diff
@@ -306,11 +306,22 @@
 
 %% 11. Copy participants.tsv
 
-% Check if participants.tsv exists
-if xASL_exist(fullfile(pathStudy, 'participants.tsv'),'file')==2
-    xASL_Copy(fullfile(pathStudy, 'participants.tsv'),fullfile(pathLegacy, 'Participants.tsv'));
-elseif xASL_exist(fullfile(pathStudy, 'Participants.tsv'),'file')==2
-    xASL_Copy(fullfile(pathStudy, 'Participants.tsv'),fullfile(pathLegacy, 'Participants.tsv'));
+% Define file names
+pathParticipantsTSV = fullfile(pathStudy, 'participants.tsv');
+pathParticipantsTSVxASL = fullfile(pathLegacy, 'participants.tsv');
+
+% Backwards compatibility: rename to lowercase
+fileListParticipantsTSVold = xASL_adm_GetFileList(pathStudy,'^Participants.tsv$',false);
+if ~isempty(fileListParticipantsTSVold)
+    pathParticipantsTSVold = fullfile(pathStudy, 'Participants.tsv');
+    pathParticipantsTSVoldTemp = fullfile(pathStudy, 'Participants_temp.tsv');
+    xASL_Move(pathParticipantsTSVold,pathParticipantsTSVoldTemp);
+    xASL_Move(pathParticipantsTSVoldTemp,pathParticipantsTSV);
```

Comment:
Also - if both old and new exist - you might decide to leave the old one, do nothing with it and report a warning. 

No likely that people have both these files, but something can go wrong during the move etc.

Reply:
At least on windows this is not even possible TBH

Reply:
And I'm not using xASL_Move with overwrite on (`xASL_Move(SrcPath, DstPath, bOverwrite, bVerbose)`), so it shouldn't overwrite the new version on default.

Reply:
Makes total sense - please add a comment in the code to avoid future 'fixes'